WPMedia/fusion-news-theme-blocks
# TMEDIA-736 : Share Bar Block

## Description

1. Share bar has been updated to new architecture.
2. Testing has been updated from Mocha to React Testing Library (100% coverage).
3. Storybook token files has been updated with block styles.
4. Feature Pack update PR can be found at https://github.com/WPMedia/arc-themes-feature-pack/pull/263 .

## Jira Ticket

- [TMEDIA-736 : Share Bar block](https://arcpublishing.atlassian.net/browse/TMEDIA-736)

## Acceptance Criteria

No criteria listed on ticket. See Description section.

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-736-share-bar-block`
2. Start Storybook with `npm run storybook`.
3. Verify both functionality and cosmetics of Share Bar block. Block will not display on mobile but will display on desktop.
<img width="269" alt="image" src="https://user-images.githubusercontent.com/452134/171277967-d4790403-114c-4d42-86bf-36f5719f3e13.png">
5. Verify test coverage of branch as well as lint compliance. 

## Effect Of Changes

Block has been updated to new architecture and styling.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

  - [X] Ran `npm run test`, made sure all tests for this branch are passing
        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
